portfolio/src/main/webapp/script.js

```diff
@@ -172,11 +182,21 @@ function constructCommentNode(comment) {
     return commentNode;
 }
 
-/** Deletes Comments */
+/** Deletes Comments 
 function deleteComments() {
-  const request = new Request("/comments", {method: 'DELETE'});
-  fetch(request).then(() => {
-    const maxComments = document.getElementById('maxcomments').value;
-    getComments(maxComments)
-  });
+    const request = new Request("/comments?id=all", {method: 'DELETE'});
+    fetch(request).then(() => {
+        const maxComments = document.getElementById('maxcomments').value;
+        getComments(maxComments);
+    });
+} */
```

Comment:
Go ahead and delete this. It'll always be there in your repo history.

Reply:
Sounds good!